### DISCUSSION TITLE
MongoDB's Application data platform [#PCIyY8rH]

### DISCUSSION THREAD

**Discussion by cyborg-ts** [terrastruct/awesome-diagrams#11] [Diagram discussion]
Autogenerated discussion for diagram at mongodbs-application-data-platform.

Insert the text "N stars" (replacing N for a number between 1 and 5) will be interpreted as a review on this diagram. Only the first review by every author will count.

**Comment by TooManyLeaves** [comment]
5 stars

How was it created?
